lib/rules/time-no-imperceptible/README.md

```diff
@@ -1,5 +1,7 @@
 # time-no-imperceptible
 
+***Deprecated: this rule is outside the scope of stylelint's functionality. See [the release planning docs](https://stylelint.io/user-guide/release-planning/) for details.***
```

Comment:
But technically new rule isn't here yet. What if it wouldn't be here when these changes will be released?